src/main/java/it/gov/pagopa/payment/dto/qrcode/TransactionCreationRequest.java

```diff
@@ -12,11 +17,15 @@
 @Builder
 public class TransactionCreationRequest {
 
+  @NotBlank(message = "This field is mandatory")
```

Comment:
it could be interesting to see what message Spring would use if none is defined here and in the @NotNull
If the default message is ok, we could remove this customization in order to not use it in each mandatory fields (otherwise there is the risk that some @NotNull fields come with a message and others with an other message)